code/modules/surgery/organ_manipulation.dm

```diff
@@ -11,6 +11,11 @@
 		/datum/surgery_step/manipulate_organs/internal,
 		/datum/surgery_step/close,
 	)
+	///Tracks if inserting or extracting an organ.
+	///Can be equal to "extract" or "insert".
+	var/current_type
```

Comment:
tie `"extract"` and `"insert"` into global string defines. easier to copy-paste and way more clear since you can DMDoc those.